atom/common/api/atom_bindings.cc

```diff
@@ -51,35 +51,43 @@ AtomBindings::~AtomBindings() {
   uv_close(reinterpret_cast<uv_handle_t*>(&call_next_tick_async_), nullptr);
 }
 
+// static
+void AtomBindings::BindProcess(v8::Isolate* isolate,
```

Comment:
Add a comment here that these bindings are shared between sandboxed & unsandboxed renderers